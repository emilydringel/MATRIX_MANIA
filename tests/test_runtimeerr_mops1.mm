def int main(){
    matrix<int> m1 = [1,2,3;4,5,6];
    matrix<int> m2 = [6,5,4;3,2,1];
    printm(m1*m2);
}