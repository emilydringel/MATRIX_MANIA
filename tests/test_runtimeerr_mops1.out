RUNTIME ERROR: matrices being multiplied do not have complementary dimensions.
